auth/client/src/main/java/org/wildfly/security/auth/client/ElytronMessages.java

```diff
@@ -162,4 +162,7 @@ ConfigXMLParseException xmlUnableToIdentifyProvider(@Param Location location, St
     @Message(id = 9527, value = "Invalid credential store reference")
     ConfigXMLParseException xmlInvalidCredentialStoreRef(@Param Location location);
 
+    @Message(id = 1167, value = "Unsupported algorithm \"%s\" for %s type")
+    ConfigXMLParseException xmlUnsupportedAlgorithmForType(@Param Location location, String algorithm, String type);
```

Comment:
The message ID 1167 has already been used elsewhere: -

./auth/jaspi/src/main/java/org/wildfly/security/auth/jaspi/_private/ElytronMessages.java:    @Message(id = 1167, value = "Unable to construct provider '%s'.")
./auth/jaspi/target/generated-sources/annotations/org/wildfly/security/auth/jaspi/_private/ElytronMessages_$logger.java:        return "ELY01167: Unable to construct provider '%s'.";

Reply:
Fixed